src/plat/bcm2711/overlay-rpi4.dts

```diff
@@ -21,4 +22,25 @@
 		/* This is configurable in the Pi's config.txt, but we use 128MiB of RAM by default. */
 		reg = <0x00 0x00000000 0x08000000>;
 	};
+
+	reserved-memory {
+		#address-cells = <0x01>;
```

Comment:
The upstream definition of this node in tools/dts/rpi4.dts uses a different definition of this property:
```
	reserved-memory {
		#address-cells = <0x02>;
```
It would probably be safer to use the same encoding here to avoid the child nodes from the original file from being misinterpreted?

Reply:
Thanks for noticing, this is just a typing error. Will fix

Reply:
Just noticed that this definition is updated on cb8c87c to be correct.